emacs/merlin.el

```diff
@@ -629,6 +632,53 @@ argument (lookup appropriate binary, setup logging, pass global settings)"
                                           (merlin-switch-list-by-ext '(".mli" ".ml")))))
   (merlin-switch-to name '(".mli" ".ml")))
 
+;;;;;;;;;;;;;;;;;;
+;; ERROR BUFFER ;;
+;;;;;;;;;;;;;;;;;;
+
+(defun merlin--error-is-short (text)
```

Comment:
I don't see that function used anywhere.

Reply:
Yes, you are right. It is a remainder of when the buffer was automatically displayed when the error was too big to fit in the minibuffer. But that turned out to be annoying, so I removed it.

This function can be safely removed.

Sorry about that oversight! I'm too used to OCaml static analysis...

Reply:
Done in 5e25357d894a8b9935d47bb64283cf9583c6818d